packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -205,9 +206,22 @@ export class OdspDocumentService implements IDocumentService {
             const websocketTokenPromise = requestWebsocketTokenFromJoinSession
                 ? Promise.resolve(null)
                 : this.getWebsocketToken!(options);
+            const joinSessionPromise = this.joinSession(requestWebsocketTokenFromJoinSession).catch((e) => {
```

Comment:
what is `e` and where does it come from?
I think it's result of createOdspNetworkError() function, and we do not propagate through Response that contains actual body of error (as it's not safe to be logged into telemetry).